requirements.txt

```diff
@@ -1,2 +1,3 @@
 PyYAML==3.11
+yapf==0.10.0
```

Comment:
This is not a dependency! Move to setup_requires (or equiv)

https://pip.pypa.io/en/latest/reference/pip_install/#controlling-setup-requires